snapcraft/tests/test_plugin_make.py

```diff
@@ -141,3 +142,19 @@ def test_build_install_var(self, run_mock):
             mock.call(['make', 'install',
                        'PREFIX={}'.format(plugin.installdir)])
         ])
+
+    @mock.patch.object(make.MakePlugin, 'run')
+    def test_build_artifacts(self, run_mock):
+        self.options.artifacts = ['artifact1', 'artifact2']
+        plugin = make.MakePlugin('test-part', self.options,
+                                 self.project_options)
+        os.makedirs(plugin.sourcedir)
+
+        plugin.build()
+
+        self.assertEqual(3, run_mock.call_count)
+        run_mock.assert_has_calls([
```

Comment:
Well, the file doesn't exist. Everything is being mocked out, which makes sense - the goal of this unit test isn't to test link_or_copy. I'll change the unit test to make sure link_or_copy is being called.